packages/db/index.d.ts

```diff
@@ -1,17 +1,23 @@
 /* eslint-disable @typescript-eslint/triple-slash-reference */
-/// <reference types="@platformatic/sql-mapper" />
 /// <reference types="@platformatic/sql-graphql" />
 /// <reference types="@platformatic/sql-openapi" />
-import ConfigManager from '@platformatic/config'
 import { FastifyInstance } from 'fastify'
 import { PlatformaticDB } from './config'
+import { PlatformaticApp as _PlatformaticApp } from '@platformatic/service'
+import { SQLMapperPluginInterface, Entities as _Entities, EntityHooks as _EntityHooks, Entity as _Entity } from '@platformatic/sql-mapper'
+import { SQLEventsPluginInterface } from '@platformatic/sql-events'
+import { DBAuthorizationPluginInterface } from '@platformatic/db-authorization'
 
-declare module '@platformatic/types' {
-  interface PlatformaticApp {
-    configManager: ConfigManager<PlatformaticDB>
-    config: PlatformaticDB
-  }
-}
+export type PlatformaticDBMixin<T extends Entities> =
+  SQLMapperPluginInterface<T> &
+  SQLEventsPluginInterface &
+  DBAuthorizationPluginInterface
+
+export type Entities = _Entities
+export type EntityHooks<T> = _EntityHooks<T>
+export type Entity<T> = _Entity<T>
+export type PlatformaticDBConfig = PlatformaticDB
+export type PlatformaticApp<T> = _PlatformaticApp<T>
```

Comment:
Instead of importing `as` and then `export`, could we simply [re-export](https://www.typescriptlang.org/docs/handbook/modules.html#re-exports) using `export ... from 'module'` ?

Reply:
good spot, thx!